plugins/build-your-own.rst

```diff
@@ -4,7 +4,7 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
```

Comment:
:+1: I will add it.

One more though: we probably have to always mention the package as well, since the versioning is not aligned between repos.